scripts/jest/setupPersistent.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
```

Comment:
Do we need both these identical files? setupPersistent and setupTests.persistent. What's the difference?

Reply:
Idk I copy-pasted the other test entry points

Reply:
Oh no wait, that's a rename mistake. I'll delete whichever one is superfluous.